governance/pyth_staking_sdk/src/utils/vesting.ts

```diff
@@ -0,0 +1,58 @@
+import type { UnlockSchedule, VestingSchedule } from "../types";
+
+export const getUnlockSchedule = (options: {
+  pythTokenListTime: bigint;
```

Comment:
You can probably handle the case where pythTokenListTime is undefined by returning an empty list maybe?

Reply:
handled it in the outer function.